mill

```diff
@@ -192,17 +55,10 @@ fi
 
 eval "$("$cs" java --env --jvm temurin:17 || "$cs" java --env --jvm openjdk:1.17.0)"
 
-
-init_mill_jvm_opts
-init_mill_cs_opts
-
 # temporary, until we pass JPMS options to native-image,
 # see https://www.graalvm.org/release-notes/22_2/#native-image
 export USE_NATIVE_IMAGE_JAVA_PLATFORM_MODULE_SYSTEM=false
 
-# clear the Mill Ammonite cache, that sometimes has spurious cached data…
-if [ -n "${CI+set}" ]; then
-  rm -rf "$HOME/.mill/ammonite/cache"
-fi
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 
-exec "$cs" launch "mill:$MILL_VERSION" "${mill_cs_opts[@]}" -- "$@"
+exec /usr/bin/env bash "$DIR/millw" "$@"
```

Comment:
NIT The `/usr/bin/env bash` shouldn't be necessary here, isn't it? (`millw` should have a `#!/usr/bin/env bash` of its own)
```suggestion
exec "$DIR/millw" "$@"
```